<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="29600" started-at="2020-02-17T15:09:39Z" finished-at="2020-02-17T15:10:08Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29600" started-at="2020-02-17T15:09:39Z" finished-at="2020-02-17T15:10:08Z">
      <class name="com.baylorgenetics.cataglog_qa.testcases.TheSearchOrdersBarOnDraft">
        <test-method status="PASS" signature="setup(org.testng.ITestContext)[pri:0, instance:com.baylorgenetics.cataglog_qa.testcases.TheSearchOrdersBarOnDraft@75881071]" name="setup" is-config="true" duration-ms="14" started-at="2020-02-17T15:09:39Z" finished-at="2020-02-17T15:09:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@13f95696]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="testTheSearchOrderBarDraftpage()[pri:0, instance:com.baylorgenetics.cataglog_qa.testcases.TheSearchOrdersBarOnDraft@75881071]" name="testTheSearchOrderBarDraftpage" duration-ms="29571" started-at="2020-02-17T15:09:39Z" finished-at="2020-02-17T15:10:08Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Could not find  in the Result page]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Could not find  in the Result page
at org.testng.Assert.fail(Assert.java:96)
at com.baylorgenetics.cataglog_qa.testcases.TheSearchOrdersBarOnDraft.testTheSearchOrderBarDraftpage(TheSearchOrdersBarOnDraft.java:227)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTheSearchOrderBarDraftpage -->
      </class> <!-- com.baylorgenetics.cataglog_qa.testcases.TheSearchOrdersBarOnDraft -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
